Smash_Combos.Domain/Models/Infraction.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Smash_Combos.Domain.Models
+{
+    public class Infraction
```

Comment:
Infraction should have `public User User` and `public User Moderator` rather than the ids